src/ytdl_sub/prebuilt_presets/helpers/premade_regex.yaml

```diff
@@ -0,0 +1,18 @@
+presets:
+
+  # Use with "YouTube Releases" to parse out each videos album and title and save accordingly
+
+  "YouTube Music Releases":
```

Comment:
I don't think this should be official - maybe in the advanced examples using regex functions. Let's exclude for now